bug_reports/Perl_JSON-Path.md

```diff
@@ -216,7 +216,7 @@ The following queries provide results that do not match those of other implement
   ```
   Actual output:
   ```
-  ["value", "entry"]
+  ["entry", "value"]
```

Comment:
Known issue with the Perl implementation being indeterministic for that query. Happy to ignore the noise for now.